spine-badge.js

```diff
@@ -25,11 +25,11 @@ import {style} from './spine-badge-css.js';
  *
  * Besides, it can be customized with the following custom properties:
  *
- * Custom property              | Description                                  | Default
- * -----------------------------|----------------------------------------------|----------
- * `--spine-badge-size`         | The height and a minimum width for the badge | `21px`
- * `--primary-background-color` | The default background for the badge         | `#ffffff`
- * `--primary-color`            | The default text color for the badge         | `rgba(0, 0, 0, 0.87)`
+ * Custom property                          | Description                                  | Default
+ * -----------------------------------------|----------------------------------------------|----------
+ * `--spine-badge-size`                     | The height and a minimum width for the badge | `21px`
+ * `--mdc-theme-background`                 | The default background for the badge         | `#ffffff`
```

Comment:
The `#fff` default value (as well as an `rgba` one below) is encapsulated in the MDC library, it's not ours. What about reflecting this somehow? E.g. we can consider something like `@polymer/theme: $mdc-theme-background`

Reply:
As discussed, the fact of using these MDC's constants look as our element's implementation detail since we embed the compiled CSS (that contains these values from MDC) as part of our component anyway.